airbyte-cdk/python/airbyte_cdk/sources/declarative/stream_slicers/substream_slicer.py

```diff
@@ -127,7 +127,7 @@ def stream_slices(self, sync_mode: SyncMode, stream_state: StreamState) -> Itera
                 stream_state_field = parent_stream_config.stream_slice_field
                 for parent_stream_slice in parent_stream.stream_slices(sync_mode=sync_mode, cursor_field=None, stream_state=stream_state):
                     empty_parent_slice = True
-                    parent_slice = parent_stream_slice.get("slice")
+                    parent_slice = parent_stream_slice
```

Comment:
this is a problem for sub-resources with an incremental parent stream.
this bug sneaked in during the stream slicer refactor